packages/core/src/error.ts

```diff
@@ -188,10 +188,6 @@ class Neo4jError extends GQLError {
     this.retriable = _isRetriableCode(code)
   }
 
-  toString (): string {
-    return this.name + ': ' + this.message
-  }
```

Comment:
why?

Reply:
This was an addition made when it seemed that the toString was broken, when that was solved this custom toString was not necessary.